content/docs/started/getting-started-gke.md

```diff
@@ -217,6 +234,11 @@ Follow these steps to deploy Kubeflow:
      The value of this variable becomes the name of your deployment.
      The contents of this directory are described in the next section.
    * **${PROJECT}** - the _name_ of the GCP project where you want kubeflow deployed to.
+   * During `kfctl init` you need to choose to use **EITHER** IAP or basic auth.
+   * `kfctl generate all` will try to fetch user email address from your 
+     credential. If it can't find a valid email address, you need to pass a
+     valid email address with flag `--email <your email address>`. This email 
+     address will be admin of your deployment.
```

Comment:
This user will be configured to be an admin of your kubeflow deployment

Reply:
done.